packages/webamp/js/components/PlaylistWindow/index.tsx

```diff
@@ -7,7 +7,7 @@ import * as Selectors from "../../selectors";
 
 import { clamp } from "../../utils";
 import DropTarget from "../DropTarget";
-import Visualizer from "../Visualizer";
+import Vis from "../Vis";
```

Comment:
Can/should we just delete the old Visualizer component? Or just replace it with the new implementation? I'm not attached to keeping it in the code base if it's not used anymore.

Reply:
I think it makes more sense to get rid of the old Visualizer component